backup/setup/migrations.json

```diff
@@ -192,6 +161,17 @@
 	},
 	"__v": 0
 },
+{
+	"_id": {
+		"$oid": "5d26e8c1d0c346ac5bf59bb5"
+	},
+	"state": "up",
+	"name": "schoolyear_19_20_added",
+	"createdAt": {
+		"$date": "2019-07-11T07:44:01.252Z"
+	},
+	"__v": 0
+},
```

Comment:
Please remove this and all other unrelated changes to the migration collection from this PR